app/controllers/application_controller.rb

```diff
@@ -30,11 +30,7 @@ def current_format
   end
 
   def formats_user_can_access
-    if current_user.gds_editor?
-      document_types
-    else
-      Hash(document_types.select { |_k, v| v.organisations.include?(current_user.organisation_content_id) })
-    end
+    Hash(document_types.select { |_, v| policy(v.klass).index? })
```

Comment:
You don't need the `Hash()` constructor around this because [`Hash#select`](http://ruby-doc.org/core-2.2.4/Hash.html#method-i-select) already returns a `Hash`.  (I think in older versions of ruby it relied on `Enumerable` for this so it got transmuted to an `Array`, but happily no longer!)